src/lib/logger/js-logger.js

```diff
@@ -13,13 +13,13 @@ const consoleHandler = Logger.createDefaultHandler({ formatter: noop })
 
 const formatter = (messages, context) => {
   const { name, level } = context
+  const originalMessages = clone(messages)
 
   if (name) {
     messages.unshift({ from: name })
   }
 
-  // log arguments was passed to event handlers as spread array
-  // e.g. log('error', a, b, c) => (context, a, b, c) not ([context, a, b, c])
+  emitter.emit('*', { name, level, messages: originalMessages })
```

Comment:
why was this changed?

Reply:
to be able also to listen for log message with any level (now we have just log.on('error|warn|... etc)